simulate/simulateIntergenic.go

```diff
@@ -204,3 +214,46 @@ func SimulateWithIndels(fastaFile string, branchLength float64, propIndel float6
 
 	return answer
 }
+
+// changebaseTransitionBias substitutes an input base b following the K80 model with a transitionBias parameter gamma.
+func changeBaseTransitionBias(b dna.Base, gamma float64) dna.Base {
+	var rand = rand.Float64()
+	switch dna.ToUpper(b) {
+	case dna.A:
+		if rand < 1.0/(2.0+gamma) { //p(a -> c) = 1 / (2 + gamma) (transversion)
+			return dna.C
+		} else if rand < 2.0/(2.0+gamma) { //p(a -> t) = 1 / (2+gamma) (transversion)
```

Comment:
The code keeps calculating:
1.0/(2.0+gamma)
and
2.0/(2.0+gamma)
very minor, but might be best to do each of those calculations once and store each in a variable.  Then all your case/if statements just reference on of those two variables.  That can also help avoid typos in one place.